### PR TITLE
[2853] Add docs for the SelectedReactionsMenu component

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://github.com/GetStream/stream-chat-android/issues/2853

### 🎯 Goal

Add docs for the `SelectedReactionsMenu` component.

### 🎨 UI Changes

screencapture-localhost-3000-chat-docs-sdk-android-compose-message-components-selected-reactions-menu-2022-01-04-19_22_20

### 🧪 Testing

1. Run the `npx stream-chat-docusaurus -i -s` command
2. Navigate to a page with the docs for the `SelectedReactionsMenu` component

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-team or #compose-chat-sdk-team) (required)
- [x] PR targets the `develop` branch

#### Code & documentation
- [x] Changelog is updated with client-facing changes
~~- [ ] New code is covered by unit tests~~
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (skydoves) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:75]: It seems to indentation was broken.
```suggestion
onDismiss = { listViewModel.removeOverlay() }
```

**Review by skydoves — APPROVED** [review]
Looks great to me! 👍

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:60]: I personally don't like this construct, we seem to be using it in quite a few places.

I think sentences such as "Handler used when..." imply causality better than "Handler for the user..."

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:61]: "Handler **used/ called** when..." might sound better.

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:82]: Not sure if the comma in "...to remove the overlay from the screen**,** in `onDismiss()`." is necessary.

Same for "It's important to note that `onMessageAction()` calls `removeOverlay()` internally**,** to hide the overlay."

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:98]: "to" is omitted in the docs.

"Allows you **to** customize"

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:100]: The can probably be omitted before "the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx)'s `reactionTypes` parameter." as it's a possessive .

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:108]: When we try to run this code we're missing a message.

Perhaps we could either add it or write "..., // State" to denote something is missing.

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:146]: I think our newer unofficial guidelines are to avoid mentioning exact implementations like these since they are inside of `DefaultX`. 

Using "reactions options" instead of `ReactionOptions` might be advisable.

Please double check this with @filbabic

File: docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-menu.mdx
Comment (MarinTolic) [inline docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-menu.mdx:149]: I think our newer unofficial guidelines are to avoid mentioning exact implementations like these since they are inside of `DefaultX`. 

Using "reactions options" instead of `ReactionOptions` might be advisable.

Please double check this with @filbabic

**Review by MarinTolic — CHANGES_REQUESTED** [review]
Requested changes only  in relation to adding the "to" on line 98 in 07-selected-reactions-menu.mdx

Everything else is purely an opinion and up to your discretion.

Great job as always @bychkovdmitry  :)

### ☑️Reviewer Checklist
- [x] UI Components sample runs & works
- [x] Compose sample runs & works
- [x] ~~UI Changes correct (before & after images)~~
- [ ] ~~Bugs validated (bugfixes)~~
- [ ] ~~New feature tested and works~~
- [x] Release notes and docs clearly describe changes
- [ ] ~~All code we touched has new or updated KDocs~~

**Review by filbabic — COMMENTED** [review]
One small thing:

In "Customization" can we please add "modifier", "shape" and "overlayColor" in the list of params for modification.

Other than that, I'll approve this and once it's resolved we can merge!

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (bychkovdmitry) [reply docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:75]: Thanks for this suggestion! I indeed used tabs here by a mistake.

File: docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-menu.mdx
Comment (filbabic) [reply docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-menu.mdx:149]: Yeah this is what we should do - have it as reaction otpions (separated, lowercase) so it doesn't show the specific component, in case we rename it., same for all other items!

File: docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx
Comment (filbabic) [reply docusaurus/docs/Android/04-compose/04-message-components/07-selected-reactions-menu.mdx:146]: Agreed

**Review by skydoves — APPROVED** [review]

**Review by filbabic — APPROVED** [review]

**Review by MarinTolic — APPROVED** [review]
Great job as always @bychkovdmitry  :)

### ☑️Reviewer Checklist
- [x] UI Components sample runs & works
- [x] Compose sample runs & works
- [x] ~~UI Changes correct (before & after images)~~
- [ ] ~~Bugs validated (bugfixes)~~
- [ ] ~~New feature tested and works~~
- [x] Release notes and docs clearly describe changes
- [ ] ~~All code we touched has new or updated KDocs~~

**Pull request merged**
